miniapp/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,10 @@
     xmlns:tools="http://schemas.android.com/tools">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+
+    <!-- Include only if your app benefits from precise location access. -->
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```

Comment:
I think you can remove these permissions, we already have those permissions in host app and we generally try not to put permissions in the sdk unless it's unavoidable.